script.js

```diff
@@ -0,0 +1,116 @@
+//interactive navigation menu
+document.addEventListener('scroll', onScroll);
+
+function onScroll(event) {
+  const curPos = window.scrollY;
+  const ancher_block = document.querySelectorAll('#body>div, section');
+  const links = document.querySelectorAll('#menu a');
+  const linksBurger = document.querySelectorAll('#menu-burger a');
+  ancher_block.forEach((el) => {
+    if (el.offsetTop <= curPos && (el.offsetTop + el.offsetHeight) > curPos) {
+      links.forEach((a) => {
+        a.classList.remove('active');
+        if (el.getAttribute('id') === a.getAttribute('href').substring(1)) {
+          a.classList.add('active');
+        }
+      })
+    }
+  });
+  ancher_block.forEach((el) => {
+    if (el.offsetTop <= curPos && (el.offsetTop + el.offsetHeight) > curPos) {
+      linksBurger.forEach((a) => {
+        a.classList.remove('active');
+        if (el.getAttribute('id') === a.getAttribute('href').substring(1)) {
+          a.classList.add('active');
+        }
+      })
+    }
+  });
+}
+
+// interactive tab menu
+const TAB = document.getElementById('tab');
```

Comment:
Why do you name it using uppercase? Please, be consistent in naming. Usually camelCase is used.

Reply:
DONE!